server.R

```diff
@@ -103,4 +103,68 @@ function(input, output, session) {
       )
     ggplotly(p)
   })
-}
\ No newline at end of file
+    
+  #********************************************************************************************************
+  # GSD start
+  #********************************************************************************************************
+  value <- reactiveValues(etaGSD = 0) # create reactive variables for dilution effect eta

```

Comment:
This is a nice technique that I didn't know. Thanks.

Reply:
I cannot take credit for this as one of my students helped me with this line. But it is indeed helpful to know.